tapis-project/tapis-ui
# Task/TUI-40 - SystemList component improvements

## Overview: ##
Provides UI and functionality improvements to the SystemList component and some of the main working areas of tapis-ui.

## Related Github Issues: ##

* [TUI-40](https://github.com/tapis-project/tapis-ui/issues/40)

## Summary of Changes: ##

- Renamed and moved Systems component to SystemList
- Replaced nav headers with common components header
- Improved UI for SystemList
- Display the selected system information under the left nav
- Set placeholder for app banner

## Testing Steps: ##
1. remove `node_modues` folder and `npm install`
2.  login with testuser
3. check Systems tab for list of systems and that they highlight when selected
4. check section headers for each area for consistency
